core/src/main/java/fr/pilato/elasticsearch/crawler/fs/FsParserAbstract.java

```diff
@@ -377,123 +384,121 @@ private void indexFile(FileAbstractModel fileAbstractModel, ScanStatistic stats,
         final long size = fileAbstractModel.getSize();
 
         logger.debug("fetching content from [{}],[{}]", dirname, filename);
-        String fullFilename = new File(dirname, filename).toString();
-
-        try {
```

Comment:
I just removed the `try finally block`.

Reply:
I see. It works because you moved up the call to `closeInputStream()`.